includes/3rd-party/themes/course.php

```diff
@@ -59,7 +59,11 @@ function sensei_load_styles_for_course_theme_variation( $variation_name ) {
 		return;
 	}
 
-	Sensei()->assets->enqueue( 'course-learning-mode-variation', 'css/3rd-party/themes/course/' . $variation_name . '.css', [] );
+	// Styles that are loaded all across when on course theme.
+	Sensei()->assets->enqueue( 'course-theme-styles', 'css/3rd-party/themes/course/style.css' );
+
+	// Variation specific styles, can override the above styles.
+	Sensei()->assets->enqueue( 'course-theme-style-variations', 'css/3rd-party/themes/course/' . $variation_name . '.css', [ 'course-theme-styles' ] );
```

Comment:
I'm not seeing the CSS in either of these files being applied. The font sizes for the elements are coming from `module.scss`. I was expecting them to be pulled from `course/default.scss` and `course/style.scss` for the Course theme.

Reply:
Hi @donnapep 👋 Just to confirm, did you checkout the Course theme branch as mentioned in the testing instructions?

> 1. Checkout this branch of Course theme https://github.com/Automattic/themes/pull/7465

In my testing, I can see it applying styles from the files mentioned above. But I can look farther if it still doesn't load for you.

Reply:
Nope, I did not. 😆 I figured I was missing something. Thanks!